src/Components/LoginPage/LoginPageComponents/SignFormInput.jsx

```diff
@@ -0,0 +1,43 @@
+import {useField} from "formik";
+import {ShowPasswordButton} from "./ShowPasswordButton";
+import React from "react";
+
+export const SignFormInput = ({label, ...props}) => {
+
+    const [field, meta] = useField(props);
+    const inputIconsClassName = {
+        firstName: "sign-bar__input_name-icon",
+        lastName: "sign-bar__input_name-icon",
+        name: "sign-bar__input_name-icon",
+        email: 'sign-bar__input_email-icon',
+        password: 'sign-bar__input_password-icon',
+        confirmPassword: 'sign-bar__input_confirm-password-icon',
+    }
+    let inputClassName = "sign-bar__input " + inputIconsClassName[props.name]
+    let passwordInputClassName = null
+
+    if (props.name === 'password' || props.name === 'confirmPassword') {
+        passwordInputClassName = "signForm_password password-block"
+
```

Comment:
Please pay attention to extra spaces in your code

Reply:
I don't see now extra space, but I try to use Ctrl+Alt+L (in WS) to refactor all spacing. 
And in feature I need to setup Linter))) to autoComplite this